api/envoy/config/route/v3/route_components.proto

```diff
@@ -920,13 +933,15 @@ message RouteAction {
     // for additional documentation.
     WeightedCluster weighted_clusters = 3;
 
-    // [#not-implemented-hide:]
-    // Name of the cluster specifier plugin to use to determine the cluster for
-    // requests on this route. The plugin name must be defined in the associated
-    // :ref:`envoy_v3_api_field_config.route.v3.RouteConfiguration.cluster_specifier_plugins`
-    // in the
-    // :ref:`envoy_v3_api_field_config.core.v3.TypedExtensionConfig.name` field.
+    // Name of the cluster provider to use to determine the cluster for requests on this route.
```

Comment:
This text still needs to be restored to the original wording.  The original wording called this a "cluster specifier plugin", which is the correct term; the modified text says "cluster provider", which is not the term we're using here.

Reply:
Get it. Thanks for calling it out.

To my surprise, I thought I had done all the name substitutions. I will check it again.